docs/VersionMigration.md

```diff
@@ -0,0 +1,36 @@
+# 1.x to 2.0 Migration Guide
+
+## Overview
+
+There are a number of breaking changes from 1.x to 2.0, mostly related to the experience trait system. If you haven't implemented any custom actions or traits, the migration will involve little or no changes in your code.
+
+## General Changes
+
+### Removed
+
+- Properties are no longer supported for anonymous users. `Appcues.anonymous(properties: Map<String, Any>?)` has been superseded by ``Appcues.anonymous()``. If you require properties, use ``Appcues.identify(userId: String, properties: Map<String, Any>? = null)``.
```

Comment:
some of these have double back ticks `` which I don't think are needed here? That may have been a carry over from DocC stuff on iOS side where it creates links?

Reply:
It doesnt change anything, I will clean this one up.